docs/api.md

```diff
@@ -180,10 +180,15 @@ Parameter      | Type         | Optional | Default | Description
 ---------      | ----         | -------- | ------- | -----------
 width          | int          | yes      | 227     | Resize images to width (`image` only)
 height         | int          | yes      | 227     | Resize images to height (`image` only)
-bw             | bool         | yes      | false   | Treat images as black & white (Caffe only)
-mean           | float        | yes      | 128     | mean pixel value to be subtracted to input image (`tensorflow` only)
-mean           | array of int | yes      | N/A     | mean pixel value per channel to be subtracted to input image (`caffe` only)
-std            | float        | yes      | 128     | standard pixel value deviation to be applied to input image (`tensorflow` only)
+bw             | bool         | yes      | false   | Treat images as black & white
+rgb	       | bool	      | yes 	 | false   | Use RGB images
+histogram_equalization | bool | yes      | false   | Whether to equalize the image histogram
+mean           | float        | yes      | 128     | mean pixel value to be subtracted to input image
+mean           | array of int | yes      | N/A     | mean pixel value per channel to be subtracted to input image
+std            | float        | yes      | 128     | standard pixel value deviation to be applied to input image
+scale	       | float	      | yes	 | 1.0	   | whether to scale the size of an image
+scale_min      | float        | yes	 | 600     | min scaling dim size
+scale_max      | float        | yes      | 1000    | max scaling dim size
```

Comment:
you can use something like `M-x align-regexp RET | RET` in emacs to make the table aligned.